src/backend/partaj/users/auth.py

```diff
@@ -2,8 +2,10 @@
 Create an authentication backend for user authenticated with Cerbère, reusing
 django-cas's built-in.
 """
-
 from django_cas_ng.backends import CASBackend
+from django_cas_ng.signals import cas_user_authenticated
+
+from . import signal
```

Comment:
If you need to import it as a side effect you can ignore the flake8 warning:

```suggestion
from . import signal  # noqa: F401
```